front/src/modules/ui/filter-n-sort/states/canPersistFiltersScopedState.ts

```diff
@@ -0,0 +1,17 @@
+import { selectorFamily } from 'recoil';
```

Comment:
this component should be in the /states/selectors subfolder
https://docs.twenty.com/developer/frontend/folder-architecture